build/pr_check_inner.sh

```diff
@@ -24,7 +24,7 @@ ls -la /dev/tty
 chmod 600 minikube-ssh-ident
 
 ssh -o StrictHostKeyChecking=no $MINIKUBE_USER@$MINIKUBE_HOST -i minikube-ssh-ident "minikube delete"
-ssh -o StrictHostKeyChecking=no $MINIKUBE_USER@$MINIKUBE_HOST -i minikube-ssh-ident "minikube start"
+ssh -o StrictHostKeyChecking=no $MINIKUBE_USER@$MINIKUBE_HOST -i minikube-ssh-ident "minikube start --cpus 6 --disk-size 36GB --memory 16000MB"
```

Comment:
I actually don't think the disk is 32G :grimacing: I believe I chose 20G

Reply:
I happen to be awake :laughing: Downsized it to 10GB